ide-support/revdeploylibraryios.livecodescript

```diff
@@ -39,16 +39,19 @@ function deployGetIphoneOSes
       end if
    else if tMacVersion < 101200 then
       put "10.2,8.2" into tList[1]
+   else if tMacVersion < 101206 then
+      put empty into tList[1]
+      answer error "To use Xcode 9.0, you need to upgrade your Mac to MacOS Sierra 10.12.6"
    else
-      // Xcode 8.3 requires a Mac running Mac OS 10.12 or later
-      put "10.3,8.3" into tList[1]
+      // Xcode 9.0 requires a Mac running Mac OS 10.12.6 or later
+      put "11.0,9.0" into tList[1]
    end if
    
    return tList
 end deployGetIphoneOSes
 
 function deployGetIosMinimumVersions
-   return "8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3"
+   return "8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,11.0"
```

Comment:
We need to keep both `10.3` and `11.0` here (it shows the deployment target in the iOS S/B settings, e.g `10.3 or later`, `11.0 or later` etc)